scripts/utilities/retry.js

```diff
@@ -3,6 +3,8 @@ function sleep(ms) {
 }
 
 function retry(functionToTry, maxAttempts = 3, delay = 1000) {
+  // We should fix this but I've not got the time at the moment
```

Comment:
Can we make this comment more useful and actionable?

Reply:
Got a suggestion in mind? If I work out how to make it actionable then I might as well just make the change and I don't really want to do that at the moment

Reply:
How about:

FIXME: reasons why this isn't good

Reply:
Have added a link to the lint rule info page which explains all that.